acme/src/main/java/io/micronaut/acme/services/AcmeService.java

```diff
@@ -138,6 +140,23 @@ public X509Certificate getCurrentCertificate() {
         }
     }
 
+    protected X509Certificate[] getFullCertificateChain() {
+        try {
+            CertificateFactory cf = CertificateFactory.getInstance("X.509");
+            File certificate = new File(certLocation, DOMAIN_CRT);
+            if (certificate.exists()) {
+                return cf.generateCertificates(Files.newInputStream(certificate.toPath())).toArray(new X509Certificate[0]);
```

Comment:
Important change was to call `cf.generateCertificates` here and then pass this into the `CertificateEvent` and thus into Netty for initialization.